source/time.tex

```diff
@@ -10795,6 +10827,18 @@
 except for white space, are parsed unchanged from the stream.
 A white space character matches zero or more white space characters in the input stream.
 
+\pnum
+If the type being parsed can not represent
+the information that the format flag refers to,
+\tcode{is.setstate(ios_base::failbit)} is called.
+\begin{example}
+A \tcode{duration} cannot represent a weekday.
```

Comment:
```suggestion
A \tcode{duration} cannot represent a \tcode{weekday}.
```

Reply:
I think this is English "weekday", not the type.

Reply:
I considered it might be so, but while checking something else in the paper, I noticed it's actually code.

Reply:
I'm going with the formatting in the LWG issues list (`\tcode` for both instances of this), though I share the concern that this might actually mean English weekday.